submissions/asaMitaka/dom/script.js

```diff
@@ -0,0 +1,201 @@
+let content = document.querySelector('.content');
+
+let aside = document.createElement('aside');
```

Comment:
Logically it's not `aside` it's `nav`. The element inside `aside` tag was excluded from the main page context(the search bot ignored them)